### PR TITLE
Adds git dotfiles, .npm-/.nvm-rc

### LINKED ISSUE #29: Add dotfiles
**Is your feature request related to a problem? Please describe.**
We should add dotfiles.

**Describe the solution you'd like**
`.gitignore`
```
**/.DS_Store
npm-debug.log*
**/node_modules
coverage
```

`.gitattributes` 
```
*.js text eol=lf
*.jsx text eol=lf
```

`.npmrc`
```
version=6.4.1
```
`.nvmrc`
```
v8.11
```

`.prettierrc`
```
 {
  "singleQuote": true,
  "trailingComma": "all"
 }
```

**Describe alternatives you've considered**
N/A

**Testing notes**

**Additional context**

- [x] Initially labelled with ["Refinement needed"](https://github.com/BBC-News/psammead/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by bcmn**
Resolves #29 

Adds basic things to gitignore, brings gitattributes in line with Simorgh.
Adds nvmrc & npmrc.

- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,2 @@
+*.js text eol=lf
+*.jsx text eol=lf
```
Comment (dr3) [inline .gitattributes:2]: Can these files have new lines added to end please?

File: .nvmrc [hunk 2]
```diff
@@ -0,0 +1 @@
+v8.11
```
Comment (dr3) [inline .nvmrc:1]: Is this a good time to move to v10?

**Review by radiocontrolled — APPROVED** [review]

File: .nvmrc (hunk 2 above)
Comment (dr3) [reply .nvmrc:1]: Talking in person, nope, need to wait for CI to catch up

**Review by dr3 — APPROVED** [review]

File: .gitattributes (hunk 1 above)
Comment (bcmn) [reply .gitattributes:2]: I think Drew & I agreed we're turning this into a separate issue, along with a Prettier rule. 😄

**Pull request merged**